### ISSUE TITLE
Mute button

### ISSUE THREAD

**Issue by maracko** [Ryochan7/DS4Windows#2507]
Hey, I am just wondering can the mute button functionality be added to dualsense drivers as is on the PS5 (press to mute internal mic or press and hold for 1 sec to disable both speaker and mic).
I know that the default driver supports it but for some reason it is disabled 99% of the time. I know that it works because when I play Destiny 2(Bungie game, owned by Sony now) the mic button magically starts working!

**Comment by Sanya192** [comment]
Mute button is implemented in current version as a special action. You can map the mute button to it, to mute the speakers.
Or you can launch a .bat file that mutes what you want.
![image](https://user-images.githubusercontent.com/11033381/187556951-cc28df28-5738-4aea-bf95-6ffeb6dd1089.png)
![image](https://user-images.githubusercontent.com/11033381/187557012-9ff8ff0b-e6c9-4461-a1db-405ea4cf6a97.png)

**Comment by maracko** [comment]
Yeah unfortunatelly that doesn't let me manipulate the internal mic/speakers, unless I am missing some info.

**Comment by Sanya192** [comment]
> Yeah unfortunatelly that doesn't let me manipulate the internal mic/speakers, unless I am missing some info.

You could write a batch script until someone implements this.
[https://superuser.com/questions/788385/is-it-possible-to-toggle-disabled-enabled-devices-from-command-line-simlar-to-d](url)

**Comment by Ryochan7** [comment]
A mute key binding would only allow muting the output volume of the currently enabled speakers. There is no direct support for muting other audio sources like a mic. Would probably need to set up an action to launch a PowerShell script or something else to toggle the mute status of the mic.

**Comment by FrodoSynthesis05** [comment]
You could set a macro for PowerToys Video Conference Mute which will allow you to choose which specific device you want to mute and then assign that macro to the mute button on DS4, I am doing that exact same thing and it works perfectly.

**Comment by maracko** [comment]
> You could set a macro for PowerToys Video Conference Mute which will allow you to choose which specific device you want to mute and then assign that macro to the mute button on DS4, I am doing that exact same thing and it works perfectly.

Thanks, this solution works great. Sadly the VCM is now in maintenance only mode

**Comment by Ansem93** [comment]
Can you add the mute botton the default buttons? 
That way who have a dualsense doesn't need to search for this topic
